index.js

```diff
@@ -1,12 +1,15 @@
 var Buffer = require('safe-buffer').Buffer
 
 module.exports = function xor (a, b) {
-  var length = Math.max(a.length, b.length)
-  var buffer = Buffer.allocUnsafe(length)
+  var min = Math.min(a.length, b.length)
+  var max = Math.max(a.length, b.length)
+  var buffer = Buffer.allocUnsafe(max)
 
-  for (var i = 0; i < length; ++i) {
+  for (var i = 0; i <= min; ++i) {
```

Comment:
won't this over-run if they are the same length?

Reply:
I was wrong, thanks for guide me.